nixos/modules/services/security/intune.nix

```
@@ -0,0 +1,32 @@
+{ config
+, pkgs
+, lib
+, ...
+}:
+with lib; let
```

```suggestion
let
```

`with lib` shouldn't be used this widely